index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="styles.css" rel="stylesheet"/>
+    <title>Challenge</title>
+  </head>
+  <body>
+ <h1>Adam Smith</h1>
+ <p id="bio">
+  Laboriosam veritatis libero facere, deleniti, suscipit eaque doloremque culpa unde cumque, consequuntur explicabo! Laboriosam nihil pariatur repudiandae doloribus reiciendis culpa perspiciatis dignissimos.
+  Eveniet mollitia in voluptas dolores placeat voluptates eum illo cupiditate vitae! Laboriosam libero nostrum ullam necessitatibus optio tempora dolorum obcaecati eligendi incidunt? Quae error animi tempora nisi repellat distinctio ducimus.
+  At est voluptatibus maxime aperiam <a href="https://www.linkedin.com" style="color:aqua blue;">link page</a> consequatur architecto, amet sapiente corrupti. Esse a reprehenderit quos, laborum, eos dignissimos quia illo reiciendis nemo sequi fugiat praesentium quasi nulla voluptates quae libero corporis!
+  Consequatur vero accusantium unde nihil soluta alias quia. Exercitationem eligendi, amet dolores excepturi enim, deleniti laudantium nesciunt perspiciatis blanditiis porro at. Temporibus architecto delectus ad cumque quas molestias reprehenderit iste.
+  Deserunt, quas doloribus ipsam et dolor earum doloremque assumenda quibusdam officiis nihil tenetur eaque debitis modi accusantium commodi. Dignissimos, saepe odio. Aspernatur at praesentium deleniti quaerat officiis minima provident voluptas.
+  Distinctio quidem sed, aliquam delectus sint officia? Nihil fuga reprehenderit sapiente temporibus, mollitia facilis molestiae quasi incidunt unde esse animi aliquam soluta aut aspernatur! Deserunt ratione consequatur sint adipisci a!
+  Eos, reiciendis. Recusandae fuga alias, error esse sint quae expedita quasi culpa consequatur corrupti facere aperiam. Fuga, aperiam. Autem excepturi quibusdam natus magnam non error cumque, assumenda rem totam debitis!
+  Consectetur adipisci eos, nulla eaque odit quis aspernatur incidunt cum dolorum beatae et, assumenda consequatur? Reiciendis, nesciunt! Labore fugit blanditiis ad incidunt eum! Debitis architecto quae ullam, commodi iste tempora.
+  Eveniet ut in, magnam quo ipsa beatae rem non ad laudantium pariatur odit hic animi qui earum quidem veritatis harum suscipit vel eos adipisci velit. Et quis voluptas sed voluptatum!
+ </p>
+ <h2>Education history</h2>
+ <div id="education">
+  <ol>
+    <li>Degree in ICT</li>
+    <li>Diploma in Business Studies</li>
+    <li>Certificate in basic computer</li>
+  </ol>
+ </div>
+  <header>Projects</header>
+  <div id="projects"></div>
+  <header>Social media</header>
+  <div id="social">
```

Comment:
Good job on using div as an HTML semantic tag. although it will be better if you use section instead of div.<section> means that the content inside is grouped, and should appear as an entry in an outline of the page. <div>, on the other hand, does not convey any meaning, aside from any found in its class, lang, and title attributes. 
**example: <section class="header"></section>**